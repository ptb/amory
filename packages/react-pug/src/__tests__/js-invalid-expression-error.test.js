import {testCompileError} from './test-helper';

testCompileError(__dirname + '/js-invalid-expression-error.input.js');
