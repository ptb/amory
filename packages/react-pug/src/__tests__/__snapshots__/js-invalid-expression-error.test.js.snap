// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect an error to be thrown 1`] = `
"<basedir>/src/__tests__/js-invalid-expression-error.input.js: Pug:2:7
    1| 
  > 2| div= 5) (10
-------------^
    3| 

Syntax Error: Unexpected token"
`;
